deployment/ansible/roles/cac-tripplanner.app/templates/upstart-cac-tripplanner-app.conf.j2

```diff
@@ -0,0 +1,14 @@
+description "Gunicorn upstart script for CAC Trip Planner"
+
+{% if develop or test -%}
+start on (vagrant-mounted)
+{% else %}
+start on (local-filesystems and net-device-up IFACE!=lo)
+{% endif %}
+stop on shutdown
+
+respawn
+setuid {{ app_username }}
+chdir {{ root_app_dir }}
+
+exec gunicorn --config {{ root_conf_dir }}/gunicorn.py cac_tripplanner.wsgi
```

Comment:
Seems to me like this could be added to azavea.gunicorn with a conditional flag or the like (and with project specific text removed). Is there a reason we don't have something like that?